### PR TITLE
Validators 1.0 branch

### PR DESCRIPTION
**Pull request by sprzenus**
Version ready for release 1.0.0

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+default_platform(:mac)
+
+platform :mac do
+
+  desc "Sets podspec version to the current tag version."
+  lane :set_podspec_version do |args|
+    version_bump_podspec(version_number: args[:version])
```
Comment (sprzenus) [inline fastlane/Fastfile:7]: I think if we really need a whole big and heavy Fastlane just for changing the version number in the pod spec. We could create a simple script for that, couldn't we?

Comment (PiotrPawlus) [reply fastlane/Fastfile:7]: Yes. We can replace Fastlane doing only this with the script. Consider doing this in the next version of if you already have prepared solution apply it to this version

File: .swiftlint.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+disabled_rules:
+  - trailing_whitespace
+  - xctfail_message
+  - trailing_comma
+
+line_length:
+  warning: 160
```
[the remark this answers is not in the document]
Comment (PiotrPawlus) [reply .swiftlint.yml:7]: Yes, please change it to 128 characters

File: fastlane/Fastfile (hunk 1 above)
Comment (sprzenus) [reply fastlane/Fastfile:7]: Ok, so I'll create an issue to v1.1 for that

Comment (PiotrPawlus) [reply fastlane/Fastfile:7]: Ok. It works for me

File: Sources/ModelValidator/Extensions/Collection+Extension.swift [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        let endingCollection = self.dropFirst(indexOffset)
+        guard !endingCollection.isEmpty else { return false }
+        return endingCollection.elementsEqual(collection)
+    }
+
+    /// Checks if the collection contains other collection in its order.
+    ///
+    /// Code from: https://forums.swift.org/t/count-of-and-contains-other-for-collection/11245
+    @inlinable
+    public func contains<T: Collection>(_ other: T) -> Bool where T.Element == Element {
```
[the remark this answers is not in the document]
Comment (PiotrPawlus) [reply Sources/ModelValidator/Extensions/Collection+Extension.swift:16]: Yes, it should be an internal function

**Pull request merged**